ticdc/ticdc-canal-json.md

```diff
@@ -129,7 +129,7 @@ TiCDC 会把一个 DDL Event 编码成如下 Canal-JSON 格式：
     ],
     "old": null,
     "_tidb": {     // TiDB 的扩展字段
-        "commitTs": 163963314122145239
+        "commitTs": 429918007904436226  // tidb tso
```

Comment:
@3AceShowHand Could you please confirm if the following documents also need to be updated:

https://github.com/pingcap/docs-cn/blob/9b974bd216e029494cad6c727a3ba67382d472f2/ticdc/ticdc-canal-json.md?plain=1#L62-L63

https://github.com/pingcap/docs-cn/blob/9b974bd216e029494cad6c727a3ba67382d472f2/ticdc/ticdc-sink-to-kafka.md?plain=1#L456-L457

https://github.com/pingcap/docs-cn/blob/9b974bd216e029494cad6c727a3ba67382d472f2/ticdc/ticdc-sink-to-kafka.md?plain=1#L522-L523

Reply:
Yes, they also need to be updated.